src/stdlib_hashmap_wrappers.fypp

```diff
@@ -87,10 +95,20 @@ end function hasher_fun
     interface get
 
         module procedure get_char_key,   &
+                         get_other, &
                          get_int8_key
 
     end interface get
 
+    interface get_other_scalar
```

Comment:
Should this go together with the `get` interface? `get_other_scalar` is a somewhat unpractical name for any application.

Reply:
It could be done, indeed.
I separated from `get` because the API of `get` is `get(input, value)` (i.e., 2 args, instead of 3 for get_other_scalar), and though that it could generate confusion in the specs.

Reply:
With [this commit ](https://github.com/fortran-lang/stdlib/pull/664/commits/5286c72b87f6a0a607f44d5b010cce1ccd8bd048) I moved it inside the `get` interface. Hopefully the specs are clear enough.